homeassistant/components/workday/binary_sensor.py

```diff
@@ -102,6 +103,25 @@ async def async_setup_entry(
                     LOGGER.debug("Removed %s by name '%s'", holiday, remove_holiday)
         except KeyError as unmatched:
             LOGGER.warning("No holiday found matching %s", unmatched)
+            async_create_issue(
+                hass,
+                DOMAIN,
+                f"bad_named_holiday-{entry.entry_id}-{slugify(remove_holiday)}",
+                is_fixable=True,
+                is_persistent=True,
```

Comment:
```suggestion
                is_persistent=False,
```

As this code is called on setup and therefore on each startup, we don't need to persistent the repair issue.